packages/ripple/package.json

```diff
@@ -22,9 +22,6 @@
   "dependencies": {
     "@leafygreen-ui/tokens": "^1.3.4"
   },
-  "devDependencies": {
-    "@leafygreen-ui/lib": "^10.0.0"
-  },
```

Comment:
This is 2 of 2 real changes. Removes unused dev dependency

Reply:
ditto here